src/components/common/DatePickers/calendar.css

```diff
@@ -17,7 +17,7 @@
 /* OVerride Date picker */
 
 .DayPicker__horizontal {
-    z-index: 2;
+    z-index: 11;
```

Comment:
Please use index, NOTE: Can do in develop

Reply:
will do while syncing with main